app/assets/stylesheets/layout_components/subscriptions.scss

```diff
@@ -0,0 +1,20 @@
+.subscriptions {
+  .row {
```

Comment:
we don't like to add custom styling to the foundation classes.
I also know that the mockups might have this much padding but we don't need that much. You might be able to apply our helper class to the html markup of p1 this will apply padding of 5px.